reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -245,11 +247,9 @@ If the objects are the same, nothing is returned.
 
 ### System.Management.Automation.PSCustomObject
 
-If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
-
-### Other
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a **SideIndicator** property to reference the differences.
 
-When you use the *PassThru* parameter, **Compare-Object** omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
+When you use the **PassThru** parameter, `Compare-Object` omits the `PSCustomObject` wrapper around the compared objects and returns the differing objects, unchanged.
```

Comment:
Don't make this a separate paragraph. Join it to the previous paragraph (no blank line between).

Reply:
okay, done.